docs/edgeql/statements/describe.rst

```diff
@@ -0,0 +1,194 @@
+.. _ref_eql_statements_describe:
+
+DESCRIBE
+========
+
+:eql-statement:
+
+``DESCRIBE`` -- provide human-readable description of a concept
+
+.. eql:synopsis::
+
+    DESCRIBE SCHEMA [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    DESCRIBE <concept-type> <name> [ AS {DDL | SDL | TEXT} ] [ VERBOSE ];
+
+    # where <concept-type> is one of
+
+      OBJECT
+      ANNOTATION
+      CONSTRAINT
+      FUNCTION
+      LINK
+      MODULE
+      PROPERTY
+      SCALAR TYPE
+      TYPE
+
+Description
+-----------
+
+``DESCRIBE`` generate a human-readable description of a schema object.
+
+The output of a ``DESCRIBE`` command is a :eql:type:`str` , although
+it cannot be used as an expression in queries. It is intended as a
+quick way to look at a piece of the database schema.
+
+There are three output formats to choose from:
+
+:eql:synopsis:`AS DDL`
+    Provide a valid :ref:`DDL <ref_eql_ddl>` definition.
+
+    The :ref:`DDL <ref_eql_ddl>` generated is a complete valid
+    definition of the particular schema object from the point of view
```

Comment:
"the point of view of the current schema" seems redundant.  Are there any other points of view?

Reply:
Yeah, like the one `AS TEXT` is using. The phrase is meant to disambiguate that the definition is from the current schema state minus this schema object, not from scratch.
Maybe I can rephrase it as "assuming all the other referenced schema objects already exist".